src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2990,8 +2987,8 @@ def webapp_up(cmd, name, resource_group_name=None, plan=None, location=None, sku
         logger.warning("Creating AppServicePlan '%s' ...", plan)
     # we will always call the ASP create or update API so that in case of re-deployment, if the SKU or plan setting are
     # updated we update those
-    create_app_service_plan(cmd, rg_name, plan, _is_linux, False, sku=sku, number_of_workers=1 if _is_linux else None,
-                            location=location)
+    create_app_service_plan(cmd, rg_name, plan, _is_linux, False, False, sku=sku,
```

Comment:
Because `per_site_scaling` is optional in your `create_app_service_plan()` with default value `per_site_scaling=False`, it will be better to specify `per_site_scaling` value here like `sku` parameter. e.g.  `create_app_service_plan(cmd, rg_name, plan, _is_linux, False, per_site_scaling=False, sku=sku, number_of_workers=1 if _is_linux else None, location=location)`